src/sentry/utils/lazy_service_wrapper.py

```diff
@@ -0,0 +1,119 @@
+"""split out from sentry.utils.services to handle mypy plugin specialization"""
```

Comment:
Feels like this should be a regular comment instead of a docstring no? since this is more implementation detail than module documentation

Reply:
yeah or probably just deleted entirely